meta-hoki/recipes-nemomobile/ngfd/ngfd/51-ffmemless.ini

```diff
@@ -0,0 +1,208 @@
+[ffmemless]
```

Comment:
I wonder if this file should be moved to meta-asteroid at some point. But not blocking this PR on that.